packages/Algernon.package/ALGWorldMenuType.class/class/menuEntries.st

```diff
@@ -0,0 +1,17 @@
+helpers
```

Comment:
Why "helpers" exactly? It seems to me that this is _strongly_ tied to the generation of the algtermlist and having the method in it's own category doesn't do code navigation any good either

Reply:
This then also applies to the other methods in this class (`menus`, `filterMenuItems`, `dockingBar`), doesn't it?
So what about putting all those in the category `algertem lists`?

Reply:
I had a look at the other types. Some put their algertem list related methods into the category `algertem list`, others put them into other categories like `helper` or `instance creation`. So I think putting them all into `algertem list` and removing the `helpers` category is fine.